### DISCUSSION TITLE
File transfer using Packet and BPSK

### DISCUSSION THREAD

**Discussion by LeonidLd** [duggabe/gr-control#15] [Q&A]
Good afternoon We use your pkt_xmt.grc scheme to transfer using HackRF. Unfortunately, we are unable to accept the transferred file; the reception results are shown in photographs. Our constellation does not diverge into two points. Don't know what could be wrong?
![image](https://github.com/user-attachments/assets/9a50a120-50f5-4722-8be5-ca5e172d8787)
![image](https://github.com/user-attachments/assets/489f4305-ae67-4a7d-a140-f290fab8f010)
![image](https://github.com/user-attachments/assets/768bd5ab-32ef-4e22-ac93-459e980b9c67)

**Comment by duggabe** [comment]
The problem probably stems from the receiver tuning as we discussed in Discussions #13

Note the issues mentioned in https://wiki.gnuradio.org/index.php?title=File_transfer_using_Packet_and_BPSK#Over_The_Air_Testing
